src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -482,6 +486,18 @@ internal void EnsureIsByRefLikeAttributeExists(DiagnosticBag diagnostics, Locati
             }
         }
 
+        internal void EnsureNullableAttributeExists(DiagnosticBag diagnostics, Location location, bool modifyCompilationForNullable)
```

Comment:
I couldn't find where we pass `modifyCompilationForNullable` as `false`. Am I missing something?

Reply:
Removed parameter from `EnsureNullableAttributeExists` and `EnsureIsByRefLikeAttributeExists`.

Reply:
The argument was not used previously but should have been used for bound lambdas. Updated.